apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -587,7 +586,8 @@ const BookingPage = ({
                   <div className="text-bookinghighlight flex items-start text-sm">
                     <FiCalendar className="ml-[2px] mt-[2px] inline-block h-4 w-4 ltr:mr-[10px] rtl:ml-[10px]" />
                     <div className="text-sm font-medium">
-                      {(rescheduleUid || !eventType.recurringEvent?.freq) && `${parseDate(date, i18n)}`}
+                      {(rescheduleUid || !eventType.recurringEvent?.freq) &&
+                        `${parseDate(date, i18n.language)}`}
```

Comment:
I made the `parseDate` function a bit easier to use by only passing in the language instead of the full i18n object. This function is moved to the features directory further down since I also had to use it in the new booker.